github/repos_releases.go

```diff
@@ -25,6 +25,7 @@ type RepositoryRelease struct {
 	Body                   *string `json:"body,omitempty"`
 	Draft                  *bool   `json:"draft,omitempty"`
 	Prerelease             *bool   `json:"prerelease,omitempty"`
+	MakeLatest             *string `json:"make_latest,omitempty"`
```

Comment:
Since there are only 3 possibilities for this parameter, would you mind documenting that here?

Something like:

```go
// MakeLatest can be one of: "true", "false", or "legacy".